modules/ToDoList.js

```diff
@@ -22,6 +22,10 @@ export default class ToDoList {
     // available tasks
     const tasks = ToDoList.getTasksList();
 
+    // const updatedTasks = tasks.array.forEach(element, elementIndex, arr => {
+    //   if(elem)
+    // });
+
```

Comment:
- [ ] Kindly take out the commented code as this would help put you in the habit of writing cleaner and better code